management/migrations/0018_auto_20170714_1233.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-07-14 19:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('management', '0017_remove_contact_contact_id'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='contact',
+            name='language_preference',
+            field=models.CharField(choices=[('ENG', 'English'), ('HIN', 'Hindi'), ('GUJ', 'Gujarati')], default='ENG', max_length=20),
```

Comment:
Yes. We do not support text sign ups in Gujarati but we do send messages out in it.